### ISSUE TITLE
Text gets centred after external image

### ISSUE THREAD

**Issue by dtu-compute** [dtu-compute/dtu-enote-website#150]
For some reason the text is centred when it is followed by an external image. See e.g. https://01005-enote.compute.dtu.dk/MarkDown-Examples

```
![Image][2]

External images can be inserted as well. [![](https://cdn.monetizejs.com/resources/button-32.png)](https://monetizejs.com/)

  [2]: https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png
```

**Comment by iainbryson** [comment]
It'll take some cleverness to fix this.  The issue is that we need to wrap the images in another element in order to center the images, and the fix for #136 leverages the fact that the markdown always generates <P> for each paragraph.  If you put a space to separate out the text, the centering won't come in:

![image](https://user-images.githubusercontent.com/1531103/28387293-7cf01e32-6c9c-11e7-9a07-6ff53beaeb3e.png)

```
External images can be inserted as well. [![](https://cdn.monetizejs.com/resources/button-32.png)](https://monetizejs.com/)

External images can be inserted as well. 
[![](https://cdn.monetizejs.com/resources/button-32.png)](https://monetizejs.com/)

External images can be inserted as well. 

[![](https://cdn.monetizejs.com/resources/button-32.png)](https://monetizejs.com/)
```

So we can either re-fix #136 by manipulating the DOM after the markdown parser and actually insert a div/p that wraps the IMG directly rather than relying on the pre-existing parent, or we can live with the space issue as it is.

But I don't think it'll be possible to have an inline image that won't be centered easily if you want other images are centered.

**Comment by dtu-compute** [comment]
Inserting a newline is perfect. Closing =)
